pkg/pointerdb/pdbclient/client.go

```diff
@@ -97,7 +99,7 @@ func (pdb *PointerDB) Get(ctx context.Context, path storj.Path) (pointer *pb.Poi
 		return nil, nil, nil, Error.Wrap(err)
 	}
 
-	pdb.authorization = res.GetAuthorization()
+	atomic.StorePointer(&pdb.authorization, unsafe.Pointer(res.GetAuthorization()))
```

Comment:
This definitely isn't correct... but I'm not sure how to fix it better yet. I think we need to remove the `SignedMessage` method and figure out what else needs to change.